invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordCitationField.js

```diff
@@ -106,20 +107,20 @@ export class RecordCitationField extends Component {
               <List celled horizontal className="separated-list">
                 {styles.map((style) => {
                   return (
-                    <List.Item>
+                    <List.Item key={style[0]}>
                       <a
-                        onClick={() => this.getCitation(record, style[0])}
-                        className={
-                          loading
-                            ? "citation-style-link loading"
-                            : "citation-style-link"
-                        }
+                        href="#"
+                        onClick={_debounce(
+                          () => this.getCitation(record, style[0]),
+                          500
+                        )}
+                        className="citation-style-link"
                       >
                         <span
                           className={
                             selectedStyle === style[0]
                               ? "selected-citation-type"
-                              : ""
+                              : "unselected-citation-type"
```

Comment:
Can you try to return here an empty string or undefined and see if it still works? If so, then we can remove the `unselected-citation-type` class.

Reply:
done